registration/migrations/0006_auto_20180105_1631.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2018-01-05 16:31
+from __future__ import unicode_literals
+
+import django.core.validators
+from django.db import migrations, models
+import registration.models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('registration', '0005_add_design_fields_v2'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='betatestapplication',
+            name='subdomain',
+            field=models.CharField(
+                error_messages={
+                    'blacklisted':
+                        'This domain name is not publicly available.',
+                    'unique':
+                        'This domain is already taken.'
+                },
+                help_text='The URL students will visit. In the future, you '
+                          'will also have the possibility to use your own '
+                          'domain name.\n\nExample: hogwarts.opencraft.hosting',
+                max_length=255,
+                unique=True,
+                validators=[
+                    django.core.validators.MinLengthValidator(
+                        3,
+                        'The subdomain name must at least have 3 characters.'
+                    ),
+                    django.core.validators.MaxLengthValidator(
+                        63,
+                        'The subdomain name can have at most have 63 '
+                        'characters.'
+                    ),
+                    django.core.validators.RegexValidator(
+                        '^[a-z0-9]([a-z0-9\\-]+[a-z0-9])?$',
+                        'Please include only lower-case letters, numbers, '
```

Comment:
nit: since your regex handles the "minimum 3 characters" requirement, I don't ever see the `'The subdomain name must at least have 3 characters.'` error.

I think it's best to keep the `MinLengthValidator` as-is, just in case someone comes along later and changes our regex.  But maybe just update the message to `"Please include at least 3 lower-case..."`